src/plfa/backmatter/Tips.md

```diff
@@ -0,0 +1,197 @@
+---
+title     : "Tips"
+permalink : /Tips/
+---
+
+# Interactive development
+
+## Equational reasoning skeleton
+
+When starting to work on an equational reasoning problem, you can begin like this:
+```agda
+begin
+    ?
+  ≡⟨ ? ⟩
+    ?
+  ≡⟨ ? ⟩
+    ?
+  ≡⟨ ? ⟩
+    ?
+∎
+```
+
+This sets up a series of reasoning steps with holes for both the expressions at each step, and the step itself.
+Since the reasoning steps are holes, Agda will always accept this, so you can make incremental progress while keeping your file compiling.
+
+Now call "solve" (`C-c C-s` in Emacs).
```

Comment:
I run `C-c C-a` (`auto`) instead, which has the added benefit of getting rid of the hole in a single step. (At least in vscode I need to `C-c C-space` to `give` after solving.)